app/validators/email_validator.rb

```diff
@@ -0,0 +1,7 @@
+class EmailValidator < ActiveModel::EachValidator
+  def validate_dach(record, attribute, value)
+    unless value =~ /\A([^@\s]+)@((?:[-a-z0-9]+\.)+[a-z]{2,})\z/i
```

Comment:
__[rubocop]__
* Use a guard clause instead of wrapping the code inside a conditional expression.